client/src/components/hooks/useStakeDeploys.js

```diff
@@ -76,7 +75,10 @@ export const useStakeFromValidators = (publicKey) => {
 		(async () => {
 			if (!publicKey) return;
 			const { data } = await dispatch(getTransferDeploysStatus(pendingStakes.map((stake) => stake.deployHash)));
-			dispatch(updateStakeDeployStatus(publicKey, 'deploys.stakes', data));
+			if (data && data.some((item) => 'success' === item.status)) {
+				dispatch(fetchValidators());
```

Comment:
I think we should alway refetch validators here.  We may have fail status or we need to update the local state if user stacked by other wallet.